src/sections/ThisYear.astro

```diff
@@ -0,0 +1,31 @@
+---
+import Button from "../components/Button";
+---
+
+<div
+  class="container flex flex-col md:flex-row items-center md:justify-between py-20 mx-auto max-w-7xl px-3 sm:px-6 md:px-10"
+>
+  <div class="flex gap-4 flex-col items-start">
+    <h2 class="text-4xl font-semibold tracking-tighter">This Year!</h2>
+    <div class="flex flex-col gap-6 mb-2 pr-20">
+      <p class="text-bg-primary text-base">
+        CMs Assemble! This year's Community Managers Africa Festival promises to
+        surpass all expectations. Attendees can look forward to engaging
+        workshops, insightful speakers, networking opportunities, and a vibrant
+        community atmosphere. Get ready for an unforgettable experience that
+        will elevate your skills and connections to new heights.
+      </p>
+    </div>
+    <Button
+      body="Buy your Tickets"
+      icon="/arrow_right_up.svg"
+      className="bg-goldenrod text-text-secondary font-semibold px-4 py-2 border-none rounded-md"
+    />
+  </div>
+  <div class="flex justify-center items-center w-full h-[520px] relative">
+    <div
+      class="flex justify-center items-end w-full h-96 bg-this-year rounded-xl mt-10"
```

Comment:
why is this image a background image? Also the width of the image and the text doesn't fit the design